invenio/modules/communities/templates/communities/index_base.html

```diff
@@ -30,6 +30,12 @@
 <div class="row">
```

Comment:
I will move the commit there later. Shall we already remove communities from 2.1?